client/src/views/ProfilePageComponents/BasicInformationForm.js

```diff
@@ -24,8 +24,10 @@ const personalInfoSchema = z.object({
     .min(1, "Last name is required")
     .regex(/^[a-zA-Z\s]+$/, "Only letters are allowed"),
   email: z.string().email("Invalid email address"),
-  phone: z.string().regex(/^\d{10}$/, "Phone number must be exactly 10 digits"),
-  expected_salary: z
+  phone: z.string().regex(/^(?!0{10})\d{10}$/, {
```

Comment:
This is not the issue. The issue is that, whenever we press next, it is validating. But if the user doesn't press next, instead if he presses overview and goes on to submit the profile, then it is still accepting. So we have to add validation when the user presses submit button as well (similar to how we are doing for next button).